index.html

```diff
@@ -2,12 +2,40 @@
 <html>
 <head>
 	<title>JS Onboard Project</title>
+	<link rel="stylesheet" type="text/css" href="style.css" >
 	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
+	
 </head>
 
 <body>
-	<p>Hello</p>
+	<div class="loader">
+		<div></div>
+	</div>
+	<div class="content">
+		<div class="controls">
+			<div class="constrols-heading">
+				<h1>IMQS Onboarding Project</h1>
+			</div>
+			<div class="constrols-search">
+				<button class="search-btn" onclick="searchFunction()">Search ID</button>
```

Comment:
You should not specify the `onclick` here, this should be done in your typescript code.